### DISCUSSION TITLE
怎么设置图表tooltip相交圆圈为空心圆？

### DISCUSSION THREAD

**Discussion by YY88Xu** [antvis/G2#6068] [Q&A] [answered]
### 问题描述

<img width="790" alt="image" src="https://github.com/antvis/G2/assets/14836228/92a160ce-d636-4f9d-b4c9-1b0a159520ff">
<img width="553" alt="image" src="https://github.com/antvis/G2/assets/14836228/fd1db760-09ee-4ca5-859c-364c366e95d2">

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by hustcc** [comment, marked as answer]
![image](https://github.com/antvis/G2/assets/7856674/5a0216e8-2d2b-4e17-9928-c0486b15df8a)

文档见这里：https://g2.antv.antgroup.com/spec/interaction/tooltip

```ts
chart.interaction('tooltip', {
  markerFill: 'opacity',
  markerStroke: 'red',
  markerLineWidth: 3,
});

```

**Reply by YY88Xu** [reply]
@hustcc markerStroke 不能设置为函数吗？空心圆的颜色跟折线图的颜色是一样的~，现在好像只能写死某个颜色
